Commonalities/documentation/CAMARA_common.json

```diff
@@ -1,612 +1,439 @@
 {
   "info": {
-    "description": "Common data and errors for Telefonica APIs",
-    "version": "0.1.0",
-    "title": "Common data and errors for Telefonica APIs",
-    "contact": {
-      "name": "API DIGITAL INTEGRATION",
-      "email": "apidigital.integration@telefonica.com"
-    }
+    "description": "Common data and errors for CAMARA APIs",

```

Comment:
@RubenBG7 : Could you please change the description and title such that it clearly states that this is just a sample?